src/ShopBundle/objects/db/TShop.class.php

```diff
@@ -656,7 +680,10 @@ public function GetCentralShopHandlerAjaxURL($sMethod, $aParameter = array())
     /**
      * overwrite the method to allow caching.
      *
-     * @return TdbShopPrimaryNaviList
+     * @return TdbPkgShopPrimaryNaviList
+     *
+     * @psalm-suppress UndefinedClass
+     * @FIXME References `TdbShopPrimaryNaviList` where `TdbPkgShopPrimaryNaviList` is probably meant
```

Comment:
This method calls `TdbShopPrimaryNaviList::GetListForShopId` - but that class does not seem to exist - maybe we want to use `TdbPkgShopPrimaryNaviList` ?